src/httpGet.tsx

```diff
@@ -0,0 +1,42 @@
+import axios, { AxiosResponse } from 'axios';
+import { URL_BASE, API_KEY, API_KEY_NUMBER } from './Data';
+import type { ResponseData } from './httpGet.types';
+
+export async function httpGet(url: string, params?: Record<string, string>) {
+    // Record -> typ Record<K, T> -> typ OBIEKTU -> gdzie: K === typ kluczy; T === typ wartości 
+    try {
+        const response: AxiosResponse<ResponseData> = await axios(
+        // zamiast pisać ręcznie - użyto AxiosResponse - typu dostarczanego przez Axios -> będącego generykiem
+        // const response: ResponseAxios = await axios(
+            {
+                baseURL: URL_BASE,
+                url,
+                params: {
+                    ...params,
+                    [API_KEY]: API_KEY_NUMBER,
+                  }
+            }
+        );
+        console.log('httpGet')
+        console.log('200');
+
+        if(params) {
+            console.log('params:');
+            console.log(params);
+        }
+        
+        console.log(response);
+        
+        return response.data.response;
+    } catch (error) {
```

Comment:
You should handle errors not here but in the UI (component using this function). Otherwise if request to the server will fail how will user know that something went wrong? You need to tell user somehow that the request failed and the data he is looking for won't be displayed.